src/app/components/category-detail/category-detail.component.ts

```diff
@@ -44,4 +64,6 @@ export class CategoryDetailComponent {
     onRemoveButtonClicked(id: string) {
         this._taskService.remove(id).subscribe(() => this._deleteSubject.next());
     }
+
+
```

Comment:
Not important for now, but in the future code like that does not look good. It's good to run formatter or just remove blank lines. Basically, you code is not formatted well. Because of that it is hard to read.